configure

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-# This script was automatically generated by ACR v1.8.1
+# This script was automatically generated by ACR v@VERSION@
```

Comment:
this is wrong, how did you installed/build acr ?

Reply:
^

Reply:
please check this

Reply:
I forgot to commit...